code/modules/shuttle/lance_docking_targeter.dm

```diff
@@ -0,0 +1,119 @@
+/obj/item/lance_docking_generator
+	name = "Lance Docking Beacon"
+	desc = "A signaling device for the thingymabober"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-red"
+	var/gps
```

Comment:
Not being used?

Reply:
oops, supposed to make the gps spawned inside it equal to that